### DISCUSSION TITLE
Change default download folder

### DISCUSSION THREAD

**Discussion by thorn11166** [LivingWithHippos/unchained-android#249] [Q&A]
Is there an option I'm missing to change default download directory? So instead of downloading to my phone it can choose a directory on my SD Card?

**Comment by LivingWithHippos** [comment]
> Is there an option I'm missing to change default download directory? So instead of downloading to my phone it can choose a directory on my SD Card?

Not at the moment since sadly the default download manager does not support it apparently. I implemented my own download manager and it's almost ready, I think you can download an alpha of the apk from the github actions section if you're feeling brave (at worst the app will crash)

**Reply by thorn11166** [reply]
I'll check it out

**Reply by LivingWithHippos** [reply]
You can test an apk in the [artifacts section here](https://github.com/LivingWithHippos/unchained-android/actions/runs/2885719077)
